src/walletSdk/Anchor/index.ts

```diff
@@ -401,6 +409,130 @@ export class Anchor {
     };
   }
 
+  /**
+  * Watch a transaction until it stops pending. Takes three callbacks:
+  * * onMessage - When the transaction comes back as pending_ or incomplete.
+  * * onSuccess - When the transaction comes back as completed / refunded / expired.
+  * * onError - When there's a runtime error, or the transaction comes back as
+  * no_market / too_small / too_large / error.
+  */
+  watchOneTransaction(params: {
+    authToken: string;
+    assetCode: string;
+    id: string;
+    onMessage: (transaction) => void;
+    onSuccess: (transaction) => void;
+    onError: (error: any) => void;
```

Comment:
error: Error | Transaction
?

Reply:
Nice catch, the type should be `Error | Transaction`

I didn't put much focus on Typing right now as we have a [dedicated task](https://stellarorg.atlassian.net/jira/software/c/projects/WAL/boards/37?modal=detail&selectedIssue=WAL-790) for it where the Transaction type should be created

I think I'll just remove the `any` Type for now and let Typescript infer it